README.md

```diff
@@ -0,0 +1,25 @@
+# Welcome
+
+This repository is the home for the content found at https://developers.eos.io/welcome
+
+MIT License
```

Comment:
I think we can link to the MIT license file in the repo instead of showing the license content here. 

https://github.com/EOSIO/welcome/blob/master/LICENSE